src/contribution-guide/continuous-integration.md

```diff
@@ -0,0 +1,16 @@
+# Continuous Integration
+
+We use GitHub Actions to build and run tests continuously.
+(But jobs without cache takes 2~3 hours to be completed)
+
+Currently (2020/10/4), we only run stdlib tests on Linux CI because the job exection time and storage reached the limit of GitHub Actions.
```

Comment:
```suggestion
Currently (2020/10/4), we only run stdlib tests on Linux CI because the job execution time and storage reached GitHub Actions limits.
```